ansible/roles/sasl/tasks/main.yml

```diff
@@ -64,7 +84,7 @@
     - role::sasl
 
 - name: Enable and start SASL authentication daemon
-  service:
```

Comment:
Because we're configuring the user and group in a way that's specific to systemd. If this service was launched with something other than systemd, then that config would silently do nothing.